services/src/main/java/org/exoplatform/addons/gamification/service/dto/configuration/RuleDTO.java

```diff
@@ -40,6 +43,7 @@ public RuleDTO(RuleEntity rule) {
         this.description = rule.getDescription();
         this.score = rule.getScore();
         this.area = rule.getArea();
+        this.domainDTO = new DomainDTO(rule.getDomainEntity());
```

Comment:
Same as previous comment

Reply:
It's already done in the second commit :)